src/scripts/farming/Plot.ts

```diff
@@ -421,11 +429,16 @@ class Plot implements Saveable {
             [MulchType.Freeze_Mulch]: GameConstants.FREEZE_MULCH_MULTIPLIER,
         }[this.mulch] ?? 1;
 
-        multiplier *= this.auraGrowth();
-
+        if (this.stage() !== PlotStage.Berry) {
+            multiplier *= this.auraGrowth();
         // Handle Death Aura
-        if (this.stage() == PlotStage.Berry && this.berry != BerryType.Kasib) {
-            multiplier *= this.auraDeath();
+        } else {
+            if (this.berry !== BerryType.Kasib) {
+                multiplier *= this.auraDeath();
+                multiplier *= this.auraDecay();
+            } else {
+                multiplier *= this.auraDecay();
```

Comment:
Move this out of your (berry is Kasib) check, then you won't have to say it twice.

Reply:
🤦 haha well i guess it's kind of obvious now that my nose isn't so close. also just noticed "death & decay".. it was the "ultimate" lich spell from warcraft III